src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj

```diff
@@ -77,11 +77,11 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.7.0-beta3" />
-    <PackageReference Include="Microsoft.ApplicationInsights.DependencyCollector" Version="2.7.0-beta3" />
-    <PackageReference Include="Microsoft.ApplicationInsights.PerfCounterCollector" Version="2.7.0-beta3" />
-    <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer" Version="2.7.0-beta3" />
-    <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" Version="2.7.0-beta3" />
+    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.7.0-beta4" />
```

Comment:
while you on it can you update `Microsoft.Extensions.DiagnosticAdapter` as well? To avoid random failures on customer apps

Reply:
let's do it separately in the next cycle (2.5) as it requires retargeting to 2.0